### PR TITLE
Add `Console.WriteLine()` in `ConsensusService.Log()`

### PR DESCRIPTION
**Pull request by erikzhang**

### CODE REVIEW

File: neo/Consensus/ConsensusService.cs [hunk 1]
```diff
@@ -175,6 +175,7 @@ private void InitializeConsensus(byte viewNumber)
 
         private void Log(string message, LogLevel level = LogLevel.Info)
         {
+            Console.WriteLine($"[{DateTime.Now.TimeOfDay:hh\\:mm\\:ss\\.fff}] {message}");
```
Comment (igormcoelho) [inline neo/Consensus/ConsensusService.cs]: I'm not against this, but do we really need this? I mean, shouldn't all logs do to Console already?

Comment (shargon) [reply neo/Consensus/ConsensusService.cs]: maybe we can change the color acording to the level?

Comment (erikzhang) [reply neo/Consensus/ConsensusService.cs]: You can change the color in another PR. 😂

Comment (vncoelho) [reply neo/Consensus/ConsensusService.cs]: aehauhea

**Pull request merged**